doc/templates/components/edit-this-page.html

```diff
@@ -0,0 +1,39 @@
+{#
+  Adapted from https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/components/edit-this-page.html
+  Required to properly link the edit button in the examples from the doc to source document in the github repo.
+#}
+
+{% extends "basic-ng/components/edit-this-page.html" %}
+{% from "basic-ng/components/edit-this-page.html" import determine_page_edit_link with context %}
+
+{%- macro furo_edit_button(url) -%}
+{%- if 'doc/auto_examples' in url -%}
+{% set url=url|replace("doc/auto_examples", "examples")|replace(".rst", ".py")|replace("index.py", "README.rst") %}
+{%- endif -%}
```

Comment:
@ymzayek 
this seemed to do the trick locally

Reply:
Just checked it and it works! Thanks a bunch

Reply:
thanks for spotting the issue and checking